config/default.yml

```diff
@@ -23,6 +23,9 @@ RSpec/ExampleWording:
     not: does not
   IgnoredWords: []
 
+RSpec/BeforeAfterAll:
+  Description: 'Do not use before/after(:all).'
```

Comment:
I think a little more explanation would be helpful, e.g. `since data created in before(:all) is not rolled back`. https://relishapp.com/rspec/rspec-rails/docs/transactions